spec/feature/hearings/add_hearing_day_spec.rb

```diff
@@ -155,4 +155,52 @@
       expect(page).to have_content(coordinator.full_name)
     end
   end
+
+  context "When adding a Virtual Hearing day" do
+    before { FeatureToggle.enable!(:national_vh_queue) }
```

Comment:
```suggestion
    before { FeatureToggle.enable!(:national_vh_queue) }
    after { FeatureToggle.disable!(:national_vh_queue) }
```
I'm not sure this is necessary, but it's standard practice.